README.md

```diff
@@ -43,23 +43,34 @@ This is quite slow and can take up to one or two hours but can provide good resu
     -   Go
     -   PHP
     -   TypeScript
+    -   Java
 
 -   Unstable (missing or incomplete unit tests)
     -   C#
+    -   Python
     -   C++
-    -   Java
     -   JavaScript
     -   Kotlin
-    -   Python
 
-### Supported Metrics
+### Supported File Metrics
+
+**complexity**<br>
```

Comment:
Let's mention in the PR that mcc was rename to complexity.

Reply:
Done